back/app/policies/initiative_policy.rb

```diff
@@ -12,13 +12,10 @@ def initialize(user, scope)
     def resolve
       not_draft = scope.where(publication_status: %w[published closed])
 
-      if user&.admin?
+      if user && UserRoleService.new.can_moderate_initiatives?(user)
         not_draft
       else
-        not_draft.left_outer_joins(:initiative_initiative_status)
-          .where.not(initiative_initiative_statuses: {
-            initiative_status_id: InitiativeStatus.where(code: InitiativeStatus::NOT_PUBLICLY_VISIBLE_CODES).select(:id)
-          })
+        not_draft.with_status_code(InitiativeStatus::CODES - InitiativeStatus::NOT_PUBLICLY_VISIBLE_CODES)
```

Comment:
Using the model scope, as discussed in a PM with Alex, but with the caveat described here.